src/content/docs/dev/send-tokens/sui/intro.mdx

```diff
@@ -79,17 +79,6 @@ Once your coin has been integrated with ITS, you can use the `send_interchain_tr
 
 The first step is to prepare the [Interchain Transfer Ticket](#interchain-ransfer-ticket) via the [prepare_interchain_transfer](#prepare-interchain-transfer) function. Once the ticket is created, you can trigger the [send_interchain_transfer](#send-interchain-transfer) function. Breaking up the transfer into a two-step process allows the package to be more flexible in the case of an upgrade.
```

Comment:
Just a note here, for convenience and usability, most txs return a `MessageTicket` that can be directly submitted to the Sui gateway contract (e.g. the ITS contract it takes care of the `prepare_interchain_transfer`) . This is true for most of the interchain txs, including `send_interchain_transfer`. 

The full list of txs which return are a `MessageTicket` that can be directly consumed by the gateway are:
- `link_coin`
- `register_coin_metadata`
- `deploy_remote_interchain_token`
- `send_interchain_transfer`